.gitignore

```diff
@@ -103,5 +103,6 @@ venv.bak/
 # mypy
 .mypy_cache/
 
-# atom remote-sync
-.remote-sync.json
+# vscode
```

Comment:
Just add `.vscode/` so anything in .vscode directory will be ignored.

Reply:
ok, made this change in the 'balanced sampler' commit